src/test/java/com/suse/saltstack/netapi/client/SaltStackClientTest.java

```diff
@@ -706,6 +714,38 @@ public void testJobsAsync() throws Exception {
     }
 
     @Test
+    public void testJobsPending() throws Exception {
+        stubFor(any(urlMatching(".*"))
+                .willReturn(aResponse()
+                .withStatus(HttpURLConnection.HTTP_OK)
+                .withHeader("Content-Type", "application/json")
+                .withBody(JSON_JOBS_RESPONSE_PENDING)));
+
+        ResultInfoSet result_set = client.getJobResult("some-job-id");
```

Comment:
Let's use mixed case with a lowercase first letter for variables here (`resultSet`) and below (`pendingMinions`).